makefile

```diff
@@ -1,4 +1,6 @@
-SRC := $(shell find src -name "*.d") \
+include makd/Makd.mak
```

Comment:
I think this should be conditioned on the existence of the `makd` sub-folder, as we don't want to make `makd` a hard dependency. Only users that want/need to use it should explicitly get it.

Reply:
good point.

Reply:
Even better, just use `makd` for packaging by doing something like:
```make
.PHONY: pkg
pkg:
        $(MAKE) -f makd/Makd.mak pkg
```

This way makd will only be used if using the `pkg` target, and will be completely unintrusive for the build process.

Reply:
@leandro-lucarella-sociomantic What do you think about obtaining makd on demand? I'm thinking something along the lines of:
```Makefile
pkg: makd
        $(MAKE) -f makd/Makd.mak pkg

makd:
    git clone https://github.com/sociomantic-tsunami/makd
```

Reply:
That way the adding makd  should be completely non-intrusive.